pkg/mobile/secrets.go

```diff
@@ -18,3 +20,17 @@ func NewSecretsCRUDL(client kubev1.CoreV1Interface) *SecretsCRUDLImpl {
 func (crudl *SecretsCRUDLImpl) Create(namespace string, secret *v1.Secret) (*v1.Secret, error) {
 	return crudl.secretsClient.Secrets(namespace).Create(secret)
 }
+
+func (crudl *SecretsCRUDLImpl) List(namespace string) (*v1.SecretList, error) {
+	listOpts := metav1.ListOptions{}
+	return crudl.secretsClient.Secrets(namespace).List(listOpts)
+}
+
+func (crudl *SecretsCRUDLImpl) Watch(namespace string) func() (watch.Interface, error) {
+	return func() (watch.Interface, error) {
+		watchOpts := metav1.ListOptions{
+			LabelSelector: "mobile",
```

Comment:
is this going to only return the secrets that have a `mobile` label?

Reply:
I hope so, haven't tested that though. will give it a try

Reply:
So this will watch only for changes in secrets with `mobile` label. But we then list all the secrets, so we still need to filter them.